flask_github.py

```diff
@@ -149,8 +233,12 @@ def raw_request(self, method, resource, params=None, **kwargs):
         if params is None:
             params = {}
 
-        if 'access_token' not in params:
-            params['access_token'] = self.get_access_token()
+        # Set ``Authorization`` header, ``access_token`` query parameter
+        if self.get_access_token():
+            params.setdefault('access_token', self.get_access_token())
```

Comment:
I believe so. I left it in there because I didn't want to remove too much of the previous code. If we got ride of that, that method would simplify quite a bit more...